### PR TITLE
Team submission + breaking down address in User schema

### PR DESCRIPTION
**Pull request by Swarita**

### CODE REVIEW

**Comment by Swarita** [comment]
Changed server\routes\api\teams.js and added a PUT request for team submission. To verify, make a PUT request in Postman and add the timings of a specific team. (Have an already created team's ID ready to add to the path: http://localhost:8082/api/teams/submit_team/:id).

File: server/routes/api/teams.js [hunk 1]
```diff
@@ -63,17 +66,39 @@ app.post('/', async (req, res) => {
...
+app.put('/team_submit/:id', async (req, res) => {
+
+  try {
+    const updatedTeam = await Team.findByIdAndUpdate(req.params.id, {
+      $set: {
+        timings: req.body.timings,
+        is_finalized: 'true'
+      }
+    }, { new: true });
+    res.send("Congratulaions! You submitted your team! Is team submitted:" + updatedTeam.is_finalized);
```
Comment (cadybaltz) [inline server/routes/api/teams.js:83]: Minor thing: there's a misspelling here

File: server/config.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+ATLAS_URI=mongodb+srv://Cluster13473:teamup@cluster13473.ziewtw1.mongodb.net/test?retryWrites=true&w=majority
```
Comment (cadybaltz) [inline server/config.js:1]: This file should not be committed to our repo - renaming to config.env should make sure it is included in our .gitignore!

**Review by cadybaltz — CHANGES_REQUESTED** [review]
Tested and works for me, see the comments for some small changes

**Review by cadybaltz — APPROVED** [review]

File: server/routes/api/registration.js [hunk 3]
```diff
@@ -29,6 +30,8 @@ app.post('/', async (req, res) => {
         email: req.body.email,
         password: req.body.password,
         address: req.body.address,
+        city: req.body.address,
```
Comment (fdolisy) [inline server/routes/api/registration.js:33]: Shouldn't this be "city: req.body.city"?

**Review by fdolisy — APPROVED** [review]

**Pull request merged**